src/main/ngapp/accio/src/app/img-gallery/img-gallery.component.html

```diff
@@ -65,16 +65,16 @@ <h1>Search by:</h1>
 <!--Filter Bar-->
 
 <!--Fetched Images-->
-<div class="gallery">
-  <div *ngFor="let image of imageArray" class="gallery-item">
-    <a [routerLink]="['/editor', projectId, image['name'], image['url']]">
+<div class="gallery" *ngIf="displayImages">
```

Comment:
What is the ngIf for? Does it actually change the behavior that the user sees, or is it the same without it? Wondering because I also had an ngIf here originally for the purposes at the time of just fetching images for the img gallery, but I removed it because there didn't seem to be any problems without it.